Detectors/TPC/qc/include/TPCQC/TrackCuts.h

```diff
@@ -0,0 +1,58 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   TrackCuts.h
+/// @author Thomas Klemenz, thomas.klemenz@tum.de
+///
+
+#ifndef AliceO2_TPC_TRACKCUTS_H
+#define AliceO2_TPC_TRACKCUTS_H
+
+#include "DataFormatsTPC/TrackTPC.h"
+
+namespace o2
+{
+namespace tpc
+{
```

Comment:
add
```
class TrackTPC;
```
here.

This should work. At least it did for me.

Reply:
Thanks, this works.